src/components/LoanFinding/QuickFiltersSection.vue

```diff
@@ -54,16 +73,42 @@ export default {
 		};
 	},
 	async mounted() {
-		this.allFacets = await fetchLoanFacets(this.apollo);
+		this.allFacets = await fetchLoanFacets(this.apollo, true);
 		await this.fetchFilterData(this.flssLoanSearch);
-	},
-	computed: {
-		totalCount() {
-			return 0;
-		},
+		const { loans, totalCount } = await runLoansQuery(
+			this.apollo,
+			{ ...this.flssLoanSearch, ...this.loanSearchState },
+			FLSS_ORIGIN_CATEGORY
+		);
+		this.loans = loans;
+		this.totalCount = totalCount;
 	},
 	methods: {
-		updateQuickFilters() {
+		async updateQuickFilters(filter) {
+			if (filter.gender !== undefined) {
+				this.flssLoanSearch.gender = filter.gender;
+			} else if (filter.sortBy) {
+				this.flssLoanSearch.sortBy = filter.sortBy;
+			} else if (filter.country) {
+				this.flssLoanSearch.countryIsoCode = filter.country;
+			} else {
+				// We want to reset the flss paramaters for categories
+				delete this.flssLoanSearch.sectorId;
+				delete this.flssLoanSearch.tagId;
+				delete this.flssLoanSearch.activityId;
+				delete this.flssLoanSearch.themeId;
+				this.flssLoanSearch = {
```

Comment:
What props do we need from `flssLoanSearch`? If there's only one prop, it would be simpler to just pull our that prop instead of delete. And we wouldn't need to add more deletes as more quick filters are added.

Reply:
a bit complicated - basically, `flssLoanSearch` determines what flss properties to apply to the flss query. This is based on the quick filter selections the user makes. We have added a new dropdown for filtering by category - however, categories can have different kinds of props associated with them for flss. Some are `themeId`, some use `sectorId`, etc. so this just resets all the props that aren't gender/sort by/location (these have specific props and hence can just be replaced when a user updates their quick filter selections) but it's more ambiguous for categories

Reply:
Got it, thanks for the info. This works for now then, but I suggest creating a follow-up ticket to re-arch this a little. For example, this component could pull from the query map instead, and be the "owner" of the filter state. That would allow this particular section of code to simply pull from the map to "reset" the state.

Reply:
Though I guess its parent could also be seen as the owner. All the to say, there's probably a way to setup this filtering path (category) for easier maintenance, since we have the query map as a point of truth, which would be simpler than having to update that filter removal list as we add filters to this quick filter feature.